### ISSUE TITLE
Accessibility: Combobox: Extra incorrect information “read only” is conveyed, when combo box list items are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3453]
Description: Extra incorrect information “read only” is conveyed, when combo box list items are accessed. 

Expected: Such extra information should not be conveyed. Only relevant information should be read out when each & every UI is accessed

Screenshot: 

![image](https://user-images.githubusercontent.com/32538291/94522394-33385800-024d-11eb-9c52-2a2c7359a75f.png)

**Comment by puru-hk** [comment]
Tested it is working as expected.
